### ISSUE TITLE
Address the payer uses to make payments should be configurable

### ISSUE THREAD

**Issue by JasoonS** [avolabs-io/flows-monorepo#32]
Maybe it should come from the graph, or pulled from the raiden-node itself?

**Comment by wynstrol** [comment]
Yeah this was one of the questions I had about the whole payment thing. Basically if you are a someone who makes payments does that mean you are always going to be running your own node? Or would they have some sort of light-client implementation specific for users who make payments?

**Comment by JasoonS** [comment]
The Payer will need to run a full node, at least in our initial versions
